bitbot/dice.go

```diff
@@ -0,0 +1,26 @@
+package bitbot
+
+import (
+	"github.com/whyrusleeping/hellabot"
+	"github.com/justinian/dice"
+	"strings"
+	"fmt"
+	
+	log "gopkg.in/inconshreveable/log15.v2"
+)
+var RollTrigger = hbot.Trigger{
+	func(irc *hbot.Bot, m *hbot.Message) bool {
+		return m.Command == "PRIVMSG"
+	},
+	func(irc *hbot.Bot, m *hbot.Message) bool {
+		roll := strings.SplitAfter(m.Content, " ")
+		rollResult, _, err := dice.Roll(roll[1])
+		if err != nil {
+			log.Error(err.Error())
+			return false  // keep processing triggers
+		}
+		resp := fmt.Sprintf("Roll: %s Result: %v", rollResult.Description(), rollResult.Int())
```

Comment:
We really only want the value of the roll itself. If you wanna get fancy, you could maybe add a check for a `verbose` option that would spit everything back into chat.